Server/controllers/parcelManger.js

```diff
@@ -1,18 +1,48 @@
+/**
+ *
+ *
+ * @class ParcelManager
+ */
 class ParcelManager {
+    /**
+     *Creates an instance of ParcelManager.
+     * @param {*} database
+     * @memberof ParcelManager
+     */
     constructor(database) {
         this.database = database;
     }
 
+    /**
+     *
+     *
+     * @param {*} packageName
```

Comment:
Lovely to see you are beginning to document your codebase. How ever, you passing in a wildcard (`*`) in place place of the types of data to expect defeats the purpose of the documentation. While this takes away the lint errors, this documentation is actually not useful for someone trying to debug your codebase, as they do not know what data types they are dealing with.
Kindly pass the appropriate data types. eg: controller methods always carry `express js` `req`(request) and `res`(response) objects so they should look something like this:
```
/**
* @param {Object} req
* @param {Object} res
* @param {String} userName
*/